source/postprocess/visualization.cc

```diff
@@ -621,12 +623,12 @@ namespace aspect
           prm.declare_entry ("Number of grouped files", "0",
                              Patterns::Integer(0),
                              "VTU file output supports grouping files from several CPUs "
-                             "into one file using MPI I/O when writing on a parallel "
+                             "into a given number of files using MPI I/O when writing on a parallel "
                              "filesystem. Select 0 for no grouping. This will disable "
                              "parallel file output and instead write one file per processor "
                              "in a background thread. "
                              "A value of 1 will generate one big file containing the whole "
-                             "solution.");
+                             "solution, while a larger value will create that many files.");
```

Comment:
what happens if you provide a number larger than the number of processors? I think it would be useful if we accepted that in which case it would mean "as many files are there are processors". Will that create trouble anywhere?